e2e/test/iothub/ConnectionStatusChangeHandlerTests.cs

```diff
@@ -126,7 +126,7 @@ void StatusChangeHandler(ConnectionStatusInfo c)
 
             // Receiving the module twin should succeed right now.
             VerboseTestLogger.WriteLine($"{nameof(IotHubDeviceClient_Gives_ConnectionStatus_Disconnected_ChangeReason_DeviceDisabled_Base)}: DeviceClient GetTwinAsync.");
-            ClientTwin twin = await deviceClient.GetTwinAsync().ConfigureAwait(false);
+            Twin twin = await deviceClient.GetTwinAsync().ConfigureAwait(false);
```

Comment:
Are we going from ClientTwin -> Twin here because we're reserving ClientTwin for the service's view of the "twin"?

Reply:
Yes, for the device client itself, it seemed more natural to use Twin here, as opposed to anywhere else.

Reply:
Renamed to TwinProperties on the device-side.